src/diffusers/schedulers/scheduling_heun_discrete.py

```diff
@@ -152,6 +168,9 @@ def set_timesteps(
         num_inference_steps: int,
         device: Union[str, torch.device] = None,
         num_train_timesteps: Optional[int] = None,
+        sigma_min: Optional[float] = None,
+        sigma_max: Optional[float] = None,
+        use_karras_sigmas: Optional[bool] = None,  # overwrite the self.config.use_karras_sigma
```

Comment:
```suggestion
```

This is not in line with our other schedulers. Note that you should define this in `__init__` `use_karras_sigmas` should only be passed to `__init__` as it's done in other schedulers